apps/dialtone-documentation/docs/components/button.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Button
 description: A button is an UI element which signals key actions to take an action throughout an app. It is important a button is identifiable, consistent, communicates its actions clearly, and is appropriately sized to its action.
-status: ready
+status: Beta
```

Comment:
button is in beta?

Reply:
Reverted it. That's when I was using it as a template for split button md.